src/docs/reports/2024/12/index.adoc

```diff
@@ -1,12 +1,22 @@
 = December 2024
 :icons: font
 
-In December 2024
+== Working Package 1 - Security of the Supply Chain
+No working
 
+== Working Package 2 - Maintenance
+* We investigated the current state of the Maven build (xref:../../../epics/77-maven-due-diligence/index.adoc[due-diligence]).
+As a first result, we identified several Maven subprojects which we could not even build, in particular the so-called https://github.com/apache/maven-sources/tree/master/aggregator[Maven reactor-build].
+* We started on working at https://github.com/support-and-care/jira-to-gh-issues[migration tool] that helps migrate Jira issues to GitHub issues. The main work was done by the Spring Team. We extended it with features, that we need for Apache Maven.
```

Comment:
Let us follow the "[one sentence per line](https://nick.groenen.me/notes/one-sentence-per-line/)" rule for AsciiDoc, Markdown, which improves maintainability as only one one line changes in Git if something is changed.